lib/std/multi_array_list.zig

```diff
@@ -252,6 +303,23 @@ pub fn MultiArrayList(comptime T: type) type {
             return index;
         }
 
+        /// Add `count` new elements at position `index`, which have undefined values.
+        pub fn addManyAt(self: *Self, allocator: Allocator, index: usize, count: usize) Allocator.Error!void {
+            try self.ensureUnusedCapacity(allocator, count);
+            return self.addManyAtAssumeCapacity(index, count);
```

Comment:
This implementation falls short of ArrayList implementation. It could be doing only 1 memcpy instead of two when the allocation must be resized. It's also missing a memset to undef after mem.copyBackwards. Please look carefully at the respective ArrayList implementations.